### DISCUSSION TITLE
HELP I MESSED UP SKIOVOX

### DISCUSSION THREAD

**Discussion by Mazedotexe** [3kh0/ext-remover#812] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
was using skiovox, got helper extension, was amazed because it is the best chrome exploit of all time 🙌

exited fullscreen using skiovox helper

somehow resized the window into oblivion or something because it disappeared

can't use skiovox anymore no matter what i do it just doesnt open, or maybe it does open but its resized to be 0x0 pixels or something

crying noises

help

**Comment by Mazedotexe** [comment]
echo please reply with something helpful lol

**Reply by 3kh0** [reply]
Hold power button and sign out, then preform the exploit again

**Comment by Mazedotexe** [comment]
> Hold power button and sign out, then preform the exploit again

@3kh0 absolutely nothing happened.

**Reply by 3kh0** [reply]
LIke did the exploit work or what? I am confused

**Reply by Mazedotexe** [reply, marked as answer]
its fixed now. the exploit didn't work, not sure why it does now but who cares tbh (im literally typing this on skiovox)

**Comment by Su4o** [comment]
@Mazedotexe You have some options. 

A. do what 3kh0 told u. (power off --> sign out button)

B. Use a different Kiosk App. 

C. Try doing the exploit again somehow.

**Reply by v4pl** [reply]
bruh

**Comment by FunnyDoggs** [comment]
If none of these work, you can press ESC+RELOAD+POWER. This will bring you to screen saying, "Please insert recovery drive or sd," or something similar. From there close the lid of the laptop, then open a couple seconds (make sure to wait!!!!) after, and then perform the exploit.
